src/go/rpk/pkg/cli/cloud/mcp/mcp.go

```diff
@@ -209,3 +197,314 @@ func newStdioCommand(fs afero.Fs, p *config.Params) *cobra.Command {
 	cmd.Flags().BoolVarP(&allowDelete, "allow-delete", "", false, "Allow delete RPCs. Off by default")
 	return cmd
 }
+
+func newProxyCommand(fs afero.Fs, p *config.Params) *cobra.Command {
+	var clusterID, mcpServerID string
+	var install bool
+	var mcpClient string
+	cmd := &cobra.Command{
+		Use:   "proxy",
+		Short: "Proxy requests to a remote Redpanda Cloud MCP server",
+		Long: `Proxy MCP requests to a remote Redpanda Cloud MCP server.
+
+This command connects to a specific cluster and MCP server running in that cluster,
+then proxies MCP requests from stdio to the remote MCP server over HTTP.
+
+Use --install to configure the MCP client instead of serving stdio.`,
+		Args: cobra.NoArgs,
+		PreRunE: func(_ *cobra.Command, _ []string) error {
+			if install && mcpClient == "" {
+				return fmt.Errorf("--client flag is required when using --install")
```

Comment:
[suggestion] flag names are used across the code, I would extract these into variables, they are used here, in L323, help text and for flag declaration.

Reply:
No problem 👍